jest.config.js

```diff
@@ -1,26 +0,0 @@
-module.exports = {
```

Comment:
thought it was loaded automatically by jest https://jestjs.io/docs/configuration

Reply:
are we not passing a custom path to the configuration file for every test we do? 

ie: https://github.com/GaloyMoney/galoy/blob/main/package.json#L20